src/emulator/hub.ts

```diff
@@ -115,7 +116,7 @@ export class EmulatorHub implements EmulatorInstance {
   }
 
   async stop(): Promise<void> {
-    this.server && this.server.close();
+    if (this.destroyServer) await this.destroyServer();
```

Comment:
nit: not a fan of one-line if statements and they're not widely used in this codebase.